<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>401</width>
                <height>447</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_21">
                            <item>
                                <spacer name="horizontalSpacer_29">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_22">
                                    <item>
                                        <widget class="QLabel" name="label_15">
                                            <property name="text">
                                                <string>姓名：</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="teacher_name_2"/>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer_30">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_23">
                            <item>
                                <spacer name="horizontalSpacer_31">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_24">
                                    <item>
                                        <widget class="QLabel" name="label_20">
                                            <property name="text">
                                                <string>籍贯：</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="teacher_home_2"/>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer_32">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_29">
                            <item>
                                <spacer name="horizontalSpacer_33">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_30">
                                    <item>
                                        <widget class="QLabel" name="label_24">
                                            <property name="text">
                                                <string>电话号码：</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="teacher_phone_2"/>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer_34">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_31">
                            <item>
                                <spacer name="horizontalSpacer_35">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_32">
                                    <item>
                                        <widget class="QLabel" name="label_16">
                                            <property name="text">
                                                <string>教师工号：</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="teacher_id_2"/>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer_36">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_33">
                            <item>
                                <spacer name="horizontalSpacer_37">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_34">
                                    <item>
                                        <widget class="QLabel" name="label_21">
                                            <property name="text">
                                                <string>从属学院：</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="teacher_college_2"/>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer_38">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_35">
                            <item>
                                <spacer name="horizontalSpacer_39">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_36">
                                    <item>
                                        <widget class="QOpenGLWidget" name="openGLWidget_2"/>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="label_25">
                                            <property name="text">
                                                <string>邮箱：</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="teacher_email_2"/>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer_40">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_37">
                            <item>
                                <spacer name="horizontalSpacer_41">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QPushButton" name="button_sure">
                                    <property name="text">
                                        <string>修改</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer_42">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
